websocket/daemon/notification.js

```diff
@@ -22,11 +22,14 @@ module.exports.display = function display(username, title, description, cb) {
     return 1;
   }
   logger.info(`${username}'s daemon is not connected`);
-  userAdapter.findByName(username).then((user) => {
-    daemonCmdAdapter.create(user.id, daemonCommandChecker.NOTIFICATION_DISPLAY, { title, description });
-  }).catch(() => {
-    logger.info(`Unable to persist notification for ${username}`);
-  });
+  if (persistence && persistence === 'true') {
```

Comment:
But if someone find this discussion someday, it may be a good thing to look at it closely